File: CEP CEMIG/testes.R
library(readxl)
library(tidyverse)

dir <- dirname(rstudioapi::getActiveDocumentContext()$path)
setwd(dir)

dados_cemig <- read_excel("Coord_Cep.xlsx")
a <- dados_cemig |> select(c("NOM_LOGRAD", "COD_CEP"))


b <- a |> group_by(COD_CEP, NOM_LOGRAD) |> summarise(n = n())
c <- b |> group_by(COD_CEP) |> summarise(n = n())

File: CEP CEMIG/testes.py
# -*- coding: utf-8 -*-
"""
Created on Tue Aug 16 11:00:59 2022

@author: michel
"""

import pandas as pd
dados_cemig = pd.read_excel('H:\FJP\CEP CEMIG\Coord_Cep.xlsx')
dados_cadunico = pd.read_excel('H:\FJP\CEP CEMIG\TESTEDADOSJUIZDEFORACEP.xlsx')


a = dados_cemig.loc[:, ['NOM_LOGRAD','COD_CEP']]

b= a.groupby( [ 'COD_CEP', 'NOM_LOGRAD'] ).size().to_frame(name = 'count').reset_index()

c = b.groupby('COD_CEP')['NOM_LOGRAD'].count()
        
       